packages/x-date-pickers/src/locales/ruRU.ts

```diff
@@ -0,0 +1,73 @@
+
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+const ruRUPickers: Partial<PickersLocaleText<any>> = {
```

Comment:
It probably does not need to be `Partial`, because all the translations are provided. 😉 
```suggestion
const ruRUPickers: PickersLocaleText<any> = {
```

Reply:
done